README.md

```diff
@@ -32,6 +32,10 @@ contributing to a specific part of the project, please join the appropriate
 special interest group (SIG). Details for all of these items are below. We are
 a friendly, collaborative group and look forward to working together!
 
+## Mission, Vision and Values
```

Comment:
apologies for the pedantic comment, but we use oxford commas throughout and should do the same here (and at otel.io/mission, for what it's worth).

I.e., `Mission, Vision, and Values`

Reply:
fixed